packages/themes/package.json

```diff
@@ -1,16 +1,16 @@
 {
   "name": "@labkey/themes",
-  "version": "1.3.1",
+  "version": "1.3.2-fb-npm-themes.0",
   "description": "Themes that come bundled with LabKey Server",
   "files": [
     "dist/"
   ],
   "scripts": {
-    "setup": "yarn install --frozen-lockfile",
+    "setup": "npm ci --legacy-peer-deps",
```

Comment:
I assume this was intentional, but just checking. ui-premium uses `npm install --legacy-peer-deps` instead of `npm ci ...`

Reply:
ah nevermind, that is for the prepublishOnly command in ui-premium